enterprise/api/v1/views/enterprise_group.py

```diff
@@ -273,6 +288,8 @@ def remove_learners(self, request, group_uuid):
                 )
                 records_deleted += len(records_to_delete)
                 records_to_delete.delete()
+                for record in records_to_delete:
```

Comment:
Do these records still exist in memory after they are deleted?

Reply:
moving this line up!